spec/models/case_comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Salesforce::CaseComment do
+  include_context "spec helper"
+
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:_case) { Fabricate(:salesforce_case, topic: topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+
+  it 'creates a case comment object on Salesforce' do
```

Comment:
Our practice is to group tests into the methods the tests are testing for. In this case it'll be `describe '#create!'`